kerastuner/engine/metrics_tracking.py

```diff
@@ -55,21 +58,26 @@ def register(self, name, direction=None):
         self.directions[name] = direction
         self.metrics_history[name] = []
 
-    def update(self, name, value):
+    def update(self, name, value, t=0):
```

Comment:
For Keras, this will be epoch number. Wanted to keep the naming as generic as possible for now as this is essentially the x-axis for any arbitrary timeseries of a tuneable process